.npmignore

```diff
@@ -0,0 +1,2 @@
+es_src
```

Comment:
`.npmignore` overrides `.gitignore`, currently it brings extra files to the packed package, for example, I got a local package about 40mb because of `node_modules` of examples. So, if `.npmignore` is used, it should contain all records from `.gitignore`. Alternatively, I'm proposing to use a `files` property of `package.json` to add only necessary folders (`dist` and `es`).